tests/phpunit/tests/media.php

```diff
@@ -2304,6 +2304,70 @@ public function test_wp_filter_content_tags_has_filter() {
 		wp_filter_content_tags( $img_tag_1 );
 		$this->assertSame( 1, $filter->get_call_count() );
 	}
+
+	/**
+	 * @ticket 55510
```

Comment:
```suggestion
	 * @ticket 55510
	 * @covers ::wp_filter_content_tags
```

:1234: Applies to the other tests too.